doc/reposync.rst

```diff
@@ -48,6 +48,9 @@ All general DNF options are accepted. Namely, the ``--repoid`` option can be use
 ``--download-metadata``
     Download all repository metadata. Downloaded copy is instantly usable as a repository, no need to run createrepo_c on it.
 
+``-g, --gpgcheck``
+    Remove packages that fail GPG signature checking after downloading. Exit code is ``1`` if at least one package was removed.
```

Comment:
I suggest we should mention here that if a specific repo has `gpgcheck=0` its packages are not checked even with this option.

Or perhaps this option should override the `gpgcheck` config value and check even packages from repos with `gpgcheck=0`?

Reply:
May be the cleanest solution would be to do the gpgcheck by default and skip it only when `--nogpgcheck` option was used. In that case `--gpgcheck` option would not be needed at all. But I do not want to change the reposync behaviour during RHEL 8 cycle.
So I will go with clarifying this in documentation.